### DISCUSSION TITLE
Not working as expected

### DISCUSSION THREAD

**Discussion by PaxVobiscuit** [Taxel/PlexTraktSync#633] [Q&A] [answered]
I used PIPX to install, and when I run plextraktsync from terminal I get the following and it appears to bomb out:

> Please enter your Plex username or e-mail: PaxVobiscuit
> If you have 2 Factor Authentication enabled on Plex you can append the code to your password below (eg. passwordCODE)
> Please enter your Plex password: 
> Login to MyPlex was successful!
> Traceback (most recent call last):
>   File "/Users/user123/.local/bin/plextraktsync", line 8, in <module>
>     sys.exit(cli())
>   File "/Users/user123/.local/pipx/venvs/plextraktsync/lib/python3.9/site-packages/click/core.py", line 1128, in __call__
>     return self.main(*args, **kwargs)
>   File "/Users/user123/.local/pipx/venvs/plextraktsync/lib/python3.9/site-packages/click/core.py", line 1053, in main
>     rv = self.invoke(ctx)
>   File "/Users/user123/.local/pipx/venvs/plextraktsync/lib/python3.9/site-packages/click/core.py", line 1659, in invoke
>     return _process_result(sub_ctx.command.invoke(sub_ctx))
>   File "/Users/user123/.local/pipx/venvs/plextraktsync/lib/python3.9/site-packages/click/core.py", line 1395, in invoke
>     return ctx.invoke(self.callback, **ctx.params)
>   File "/Users/user123/.local/pipx/venvs/plextraktsync/lib/python3.9/site-packages/click/core.py", line 754, in invoke
>     return __callback(*args, **kwargs)
>   File "/Users/user123/.local/pipx/venvs/plextraktsync/lib/python3.9/site-packages/plextraktsync/commands/plex_login.py", line 151, in plex_login
>     login(username, password)
>   File "/Users/user123/.local/pipx/venvs/plextraktsync/lib/python3.9/site-packages/plextraktsync/commands/plex_login.py", line 162, in login
>     [server, plex] = choose_server(account)
>   File "/Users/user123/.local/pipx/venvs/plextraktsync/lib/python3.9/site-packages/plextraktsync/commands/plex_login.py", line 121, in choose_server
>     click.echo(title(f"Attempting to connect to {server.name}. This may take time and print some errors."))
> AttributeError: 'NoneType' object has no attribute 'name'

**Comment by PaxVobiscuit** [comment, marked as answer]
Disregard. I was not logged in to my Plex Server and it was unclaimed.

**Comment by glensc** [comment]
Bug reports to developers go to issues, discussions are for user to user feedback
